app/src/shared/components/NotFoundPage/index.jsx

```diff
@@ -39,7 +39,7 @@ const NotFoundPage = () => (
                         <Button
                             kind="special"
                             tag={Link}
-                            to={routes.core()}
+                            to={routes.marketplace.index()}
```

Comment:
Fix marketplace link in 404 page ([PLATFORM-1400](https://streamr.atlassian.net/browse/PLATFORM-1400))